podpac/core/utils.py

```diff
@@ -541,4 +544,33 @@ def get_entry(key, out, definition):
             entry["inputs"] = {}
         return entry
 
-    return get_entry(list(out.keys())[-1], out, definition)
+    c = [(v, d) for v, d in zip([lat, lon, time, alt], ["lat", "lon", "time", "alt"]) if v is not None]
+    coords = podpac.Coordinates([[v[0]] for v in c], [[d[1]] for d in c])
+    v = float(node.eval(coords))
+    definition = node.definition
+    out = OrderedDict()
+    for key in definition:
+        if key == "podpac_version":
+            continue
+        d = partial_definition(key, definition)
+        n = podpac.Node.from_definition(d)
+        value = float(n.eval(coords))
+        inputs = flatten_list(list(d[key].get("inputs", {}).values()))
+        active = True
+        out[key] = {
+            "active": active,
+            "value": value,
+            "inputs": inputs,
+            "name": n.style.name if n.style.name else key,
+            "node_hash": n.hash,
+        }
+        # Fix sources for Compositors
+        if isinstance(n, podpac.compositor.OrderedCompositor):
+            for inp in inputs:
+                out[inp]["active"] = False
+                if out[inp]["value"] == out[key]["value"] and np.isfinite(out[inp]["value"]):
+                    out[inp]["active"] = True
```

Comment:
Hmm, I expected a break here. Only the first non-nan source is active, right?

Reply:
Yes, but by default I'm setting everything to `False`. So any layers after the `active` one still needs their default changed from `True` to `False`.

Reply:
Thanks for reviewing!